app/views/layouts/application.haml

```diff
@@ -67,8 +67,11 @@
 
   :javascript
     $(function() {
-      FastClick.attach(document.body, { excludeNode: '^pac-'} );
-      $('input, textarea').placeholder();
+      if ('touchAction' in document.body.style) {
+        document.body.style.touchAction = 'manipulation';
+      } else {
+        window.FastClick.attach(document.body, { excludeNode: '^pac-'} );
```

Comment:
FastClick & Google Autocomplete don't want to be in the same page. We had to patch FastClick while making Location search.